_data/icons/point.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4/icon.png",
```

Comment:
```
✗ ~/tmp identify bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4/icon.png 
bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4/icon.png PNG 512x512 512x512+0+0 8-bit sRGB 33448B 0.000u 0:00.000
```
the size is wrong. Also please make the CID the direct file - not wrapped in a directory